servo/connectors/kubernetes.py

```diff
@@ -3725,7 +3735,7 @@ async def components(self) -> List[servo.Component]:
         return state.to_components()
 
     @servo.before_event(servo.Events.measure)
-    async def before_measure(self) -> None:
+    async def before_measure(self, **kwargs) -> None:
```

Comment:
Why is this being updated when control and environment aren't used by this event handler?

Reply:
Because (due to a change elsewhere, which is needed to provide the event data in 'before' handlers), this event handler *will* be called with additional arguments - even though it doesn't need them. The way ServoX appears to be designed, ALL event handlers of the same kind (before, on or after) are called with the same exact args - the event dispatcher loops through all of them and calls them with the same exact args (whether the handler wants them or not).